src/test/kotlin/racingcar/service/RacingCarGameServiceTest.kt

```diff
@@ -0,0 +1,33 @@
+package racingcar.service
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.inspectors.shouldForAll
+import io.kotest.matchers.shouldBe
+
+class RacingCarGameServiceTest : BehaviorSpec({
+
+    Given("자동차 경주 게임은") {
+        val countOfRacer = 3
+        val racingCarGameService = RacingCarGameService()
+        When("처음 시작하면") {
+            racingCarGameService.init(countOfRacer)
+            Then("레이서의 수만큼 레이서가 만들어진다.") {
+                racingCarGameService.racers.count() shouldBe countOfRacer
+            }
+        }
+    }
+
+    Given("자동차 경주 게임이 한 차례 진행되면") {
+        val racingCarGameService = RacingCarGameService()
+        racingCarGameService.init(3)
+        val fixedNumGenerator = FixedNumGenerator()
+        When("모든 플레이어들은") {
+            racingCarGameService.run(fixedNumGenerator)
```

Comment:
`NumberGenerator` interface를 [SAM](https://kotlinlang.org/docs/fun-interfaces.html#sam-conversions)으로 변경하면, FixedNumberGernator 구현 없이도 테스트를 조금 더 간편하게 수행할 수 있을 것 같아요. 😃

```suggestion
        When("모든 플레이어들은") {
            racingCarGameService.run { 4 }
```